filament/backend/test/test_ReadPixels.cpp

```diff
@@ -318,4 +318,101 @@ TEST_F(BackendTest, ReadPixels) {
     getDriver().purge();
 }
 
+static bool gReadPixelsFinished = false;
```

Comment:
It would be better to write a unit test that uses a custom class+instance instead of relying on a global field.